files/jcl/ZWECSVSM.jcl

```diff
@@ -0,0 +1,57 @@
+//ZWECSVSM JOB
```

Comment:
Your IDCAMS comments will cause JCL and IDCAMS errors. Also, deleting semi-random lines based on whether you want RLS or not is asking for errors, so it's better to provide two blocks, and a simple switch between them.
I reworked the JCL and attached it here.

[ZWECSVSM.jcl.txt](https://github.com/zowe/zowe-install-packaging/files/5478894/ZWECSVSM.jcl.txt)